locust/main.py

```diff
@@ -346,7 +346,11 @@ def spawn_run_time_quit_greenlet():
     def start_automatic_run():
         if options.master:
             # wait for worker nodes to connect
+            start_time = time.monotonic()
             while len(runner.clients.ready) < options.expect_workers:
+                if options.expect_workers_max_wait and options.expect_workers_max_wait < time.monotonic() - start_time:
+                    logger.error("Gave up waiting for workers to connect.")
```

Comment:
could you add
        ```
if runner is not None:
            runner.quit()
```
here to remove all workers, if any?

Reply:
sure

Reply:
done